src/components/dialogs/line-split-with-voltage-level-dialog.js

```diff
@@ -421,20 +421,19 @@ const LineSplitWithVoltageLevelDialog = ({
                     </Grid>
                     <GridSection title="VoltageLevelToSplitAt" />
                     <Grid container spacing={2}>
-                        {gridItem(voltageLevelIdField, 5)}
+                        {gridItem(voltageLevelIdField, 6)}
+                        {gridItem(bbsOrNodeIdField, 6)}
                         {gridItem(
                             <Button
                                 onClick={openVoltageLevelDialog}
                                 startIcon={<AddIcon />}
-                                variant="outlined"
                             >
                                 <Typography align="left">
                                     <FormattedMessage id="NewVoltageLevel" />
                                 </Typography>
                             </Button>,
-                            2
+                            6
```

Comment:
6+6+6 = 18 the Grid container components accept 12 column by default, this seems strange. 6 is the default value as well for the saze parameter of gridItem then you could remove it in this case :)